### PR TITLE
feat(Script/Commands): .debug dummy

### PR DESCRIPTION
**Pull request by Kitzunu**
* cherry-pick commit (https://github.com/TrinityCore/TrinityCore/commit/5ed891a0f0c65cf4c844ccb209952352f4241193)

Co-Authored-By: Treeston <14020072+Treeston@users.noreply.github.com>

## Changes Proposed:
-  good stuff

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Review by IntelligentQuantum — APPROVED** [review]

File: src/server/scripts/Commands/cs_debug.cpp [hunk 1]
```diff
@@ -1244,6 +1245,12 @@ class debug_commandscript : public CommandScript
         handler->PSendSysMessage("Waypoint SQL written to SQL Developer log");
         return true;
     }
+
+    static bool HandleDebugDummyCommand(ChatHandler* handler, uint32 /*args*/)
```
Comment (Winfidonarleyan) [inline src/server/scripts/Commands/cs_debug.cpp:1249]: why need uint32?

Comment (Kitzunu) [reply src/server/scripts/Commands/cs_debug.cpp:1249]: I didnt know what to put for args other than uint32

Comment (Winfidonarleyan) [reply src/server/scripts/Commands/cs_debug.cpp:1249]: Then make this command without arguments

Comment (FrancescoBorzi) [reply src/server/scripts/Commands/cs_debug.cpp:1249]: @Kitzunu

Comment (IntelligentQuantum) [inline src/server/scripts/Commands/cs_debug.cpp:1249]:
```suggestion
    static bool HandleDebugDummyCommand(ChatHandler* handler)
```

**Review by Winfidonarleyan — APPROVED** [review]

**Pull request merged**